protobuf_version.bzl

```diff
@@ -1,7 +1,7 @@
 """ Contains version numbers to be used in other bzl files """
-PROTOC_VERSION = "29.3"
-PROTOBUF_JAVA_VERSION = "4.29.3"
-PROTOBUF_PYTHON_VERSION = "5.29.3"
-PROTOBUF_PHP_VERSION = "4.29.3"
-PROTOBUF_RUBY_VERSION = "4.29.3"
+PROTOC_VERSION = "29.1"
```

Comment:
Once you revert these version changes, the PR should be a lot smaller.

Reply:
Yes, without the versions it's smaller. But none of the tests pass, because code is generated with a different version.